v3/sofle_choc/sofle_choc.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Sofle Choc",
+  "vendorId": "0xFC32",
+  "productId": "0x0287",
```

Comment:
This conflicts with `sofle/rev1`

Reply:
https://github.com/qmk/qmk_firmware/blob/master/keyboards/sofle/rev1/info.json#L7-L8

Reply:
Good catch! I opened an issue with Sofle repo to see if [we can fix that duplicate ID](https://github.com/josefadamcik/SofleKeyboard/issues/194#issuecomment-1942966178).

Reply:
The sofle_choc creator, Brian Low, [approved a PID/VID change](https://github.com/josefadamcik/SofleKeyboard/issues/194#issuecomment-1943001616).
I opened a [PR with QMK to make the sofle_choc PID/VID unique](https://github.com/qmk/qmk_firmware/pull/23080#issue-2134483702).

Reply:
I goofed and closed that initial QMK PR. A new one has been opened with the same vid/pid fix:
https://github.com/qmk/qmk_firmware/pull/24030

Reply:
PR accepted - hurray!